### ISSUE TITLE
Display ownership in steam widgets

### ISSUE THREAD

**Issue by Revadike** [tfedor/AugmentedSteam#901]
It would be nice to see if I own a game from a steam widget. 

For example, these games are being given away and I own them, but I still need to check manually if I own them to be sure:
![image](https://user-images.githubusercontent.com/4411977/82695905-48312900-9c66-11ea-9db6-ebcd14cf8c87.png)

**Comment by candela97** [comment]
Same as #700?

**Comment by Revadike** [comment]
Does he mean widgets too?

**Comment by candela97** [comment]
> Does he mean widgets too?

Think so. Your title is better though, so I think we can track this feature here instead.

**Comment by candela97** [comment]
We won't be able to highlight the title or add anything to the widget itself because access to the iframe's `contentDocument` is blocked by the browser due to cross-origin restrictions. I think the only options are adding a tag or text label above the widget or using tooltips.

**Comment by Revadike** [comment]
If it's an iframe, can't you just apply the script to the iframe location itself? I guess it would be difficult to communicate owned games then, but could use postMessage for communication.

**Comment by candela97** [comment]
You mean like apply the script to https://store.steampowered.com/widget/440/?dynamiclink=1 and replace the one on the page? Hmm...

**Comment by Revadike** [comment]
Yes, run the code on that link. Not sure what needs replacing though.

**Comment by candela97** [comment]
So you mean code ran on that link will be reflected when the iframes load on the page?

**Comment by Revadike** [comment]
> So you mean code ran on that link will be reflected when the iframes load on the page?

Yes, that's the same way as [I load a script for recaptcha to click the 'I'm not a robot' checkmark](https://greasyfork.org/en/scripts/18449-recaptcha-helper) :D
